src/Main/CastEfficiency.js

```diff
@@ -12,6 +12,7 @@ const CastEfficiency = ({ categories, abilities }) => {
       <table className="data-table" style={{ marginTop: 10, marginBottom: 10 }}>
         {Object.keys(categories)
           .filter(key => abilities.some(item => item.ability.category === categories[key])) // filters out categories without any abilities in it
+          .filter(key => categories[key] !== '') //filters out the hidden category
```

Comment:
Could make this more clear by saying ` ... !== Abilities.SPELL_CATEGORIES.HIDDEN`